app/controllers/application_controller.rb

```diff
@@ -143,6 +144,15 @@ def check_edits_limit!(post)
 
   private
 
+  def distinguish_fake_community
+    if @community.is_fake
```

Comment:
Where is `@community` coming from? If you're looking for the current community, that's `RequestContext.community`.

Reply:
Both are equal, see l. 213; changed though